README.md

```diff
@@ -64,16 +64,48 @@ jekyll serve --watch --config _config.yml,_config-dev.yml
+
+## 👩‍🚀 Contributors, backers & sponsors
+
+This project exists thanks to all the **people who contribute**.
+<a href="graphs/contributors"><img src="https://opencollective.com/opensourcedesign/contributors.svg?width=890&button=false" /></a>
+
+Thank you to **all our backers**! 🙏 ([Become a backer](https://opencollective.com/opensourcedesign#backer))
+
+<a href="https://opencollective.com/opensourcedesign#backers" target="_blank"><img src="https://opencollective.com/opensourcedesign/backers.svg?width=890"></a>
+
+**Support this project by becoming a sponsor.** Your logo will show up here with a link to your website. ([Become a sponsor](https://opencollective.com/opensourcedesign#sponsor))
+
+<a href="https://opencollective.com/opensourcedesign/sponsor/0/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/0/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/1/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/1/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/2/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/2/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/3/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/3/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/4/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/5/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/6/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/7/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/8/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/9/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/9/avatar.svg"></a>
+
+
+## ♥ Code of Conduct
+
+Please note that Open Source Design has a [Contributor Code of Conduct](https://opensourcedesign.net/code-of-conduct/). By participating in this project online or at events you agree to abide by its terms.
+
+
+## 📜 License
+
+- **🔀 You can use & modify everything as long as you credit [Open Source Design](https://opensourcedesign.net) and use the same license for your resulting work.** [Code license is AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) and content is [Creative Commons Attribution-ShareAlike](https://creativecommons.org/licenses/by-sa/4.0/).
```

Comment:
I'm not really sure how easily is to change the code from one license to another; OR the differences between them, but I trust you did an informed decision :P

Reply:
Basically, Apache2 is like Creative Commons Attribution, whereas AGPL is like Creative Commons Attribution-ShareAlike, meaning remixed work has to be licensed under the same license again, having it perpetually in the open. For our website, which is mostly informative lists, this is definitely desirable?